### ISSUE TITLE
19- Matt Oliverio

### ISSUE THREAD

**Issue by ironyarder77** [TIY-Durham/2015-FALL-FEE#365]
- [x] **Yak Shaving**
  - _WIP Issue_
    - link to PRs in:
      - [x] `USERNAME.github.io`
      - [x] `TIY-Chessboard`
  - _WIP Branch_
    - [x] `USERNAME.github.io:tutorial-week4`
    - [x] `TIY-Chessboard:19--world-war-chess`
  - _WIP Files_
    - in `USERNAME.github.io:tutorial-week-4` or whatever
    - in `TIY-Chessboard:19--world-war-chess`
      - `js/`
        - `controllers.js`
        - `main.js` -- in case something breaks
        - `tests.js` -- Controllers are hard to test
      - `index.html` -- wire 'em up!
- [x] Journal, Week 4
  - [x] Final draft of tutorial written!
  - [x] 2 x :+1: reviews!
  - [x] And you give 2 x :+1:
- [x] **World War Chess**

https://github.com/TIY-Durham/2015-FALL-FEE/issues/365

**Comment by ironyarder77** [comment]
created issue for 19

**Comment by ironyarder77** [comment]
LINK TO 19--world-war-chess:
PR https://github.com/ironyarder77/TIY-Chessboard/pull/4

LINK TO JASMINE'S TUTORIAL:
https://github.com/boshnivolo/boshnivolo.github.io/commit/e2c78d3d3851fb0ad096a59862759499f7a178a9

**Comment by ironyarder77** [comment]
Have been slowly working through the chessboard assignment.
As I began to experiment with the css styles, things started really connecting for me.
My first instinct is to start designing buttons but I know I can't do that right now:)
I was able to figure out the or plot the first five moves using the rank and file method.
I was also able to create the links for the buttons and then got them to populate the display.
Although positioning them has become a pain..and I though I knew how to do this.
